src/wui/mapdetails.cc

```diff
@@ -46,18 +49,41 @@ MapDetails::MapDetails(
                  "",
                  UI::Align::kLeft,
                  UI::MultilineTextarea::ScrollMode::kNoScrolling),
-     descr_(&main_box_, 0, 0, UI::Scrollbar::kSize, 0, style, ""),
-     suggested_teams_box_(new UI::SuggestedTeamsBox(this, 0, 0, UI::Box::Vertical, padding_, 0)) {
+     descr_(&descr_box_,
+            0,
+            0,
+            UI::Scrollbar::kSize,
+            0,
+            style,
+            "",
+            UI::Align::kLeft,
+            UI::MultilineTextarea::ScrollMode::kNoScrolling),
+     minimap_icon_(&descr_box_, 0, 0, 0, 0, nullptr),
+     suggested_teams_box_(new UI::SuggestedTeamsBox(this, 0, 0, UI::Box::Vertical, padding_, 0)),
+     egbase_(nullptr) {
+
+	minimap_icon_.set_frame(g_style_manager->minimap_icon_frame());
+	descr_.set_handle_mouse(false);
+	descr_box_.set_force_scrolling(true);
+
+	descr_box_.add(&descr_, UI::Box::Resizing::kAlign, UI::Align::kLeft);
+	descr_box_.add_space(padding_);
+	descr_box_.add(&minimap_icon_, UI::Box::Resizing::kAlign, UI::Align::kCenter);
```

Comment:
Looks like there's a bug in `UI::Box` with center align when there is a scroll bar. I leave it up to you whether you want to fix that in this branch, since we currently have another PR open for the UI::Box.

Reply:
I fixed the alignment